composer.json

```diff
@@ -32,7 +32,7 @@
         }
     },
     "replace": {
-        "paragonie/random_compat": "*",
+        "paragonie/random_compat": "9.99.99",
```

Comment:
I suggest replacing a 2.x version instead. Many packages out there have not been updated yet to allow this special 9.99.99 version and they would be incompatible with the skeleton (and no package out there would have a requirement allowing *only* that version, as that would be the same than not needing it at all)

Reply:
but then when a package is ok with 9.99, the package will still be installed, isn't it?

Reply:
nope, the replace rule in the rule package creates a conflict between the root package and *any* version of the actual package (that's why we use `replace` and not `provide`) so they cannot be installed together.